.github/actions/zombienet-sdk/action.yml

```diff
@@ -21,12 +21,6 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: k8s_auth
```

Comment:
@pepoviola would be good to extract unrelated zombienet/CI changes to a separate PR so we can backport this cleanly

Reply:
Hey @ordian, Yes. We are planning to make a new `pr` to stabilize ci today.